ocp_resources/datavolume.py

```diff
@@ -212,14 +221,20 @@ def scratch_pvc(self):
             client=self.client,
         )
 
+    def _is_pending(self):
```

Comment:
Does not check pending but returns `self.instance.status.phase`
If you need `self.instance.status.phase` - use where needed, no need for a function.
If you need to check Pending state, please update the function.